spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+RSpec.describe VehicleFactory do
+
+  it 'exists' do
+    factory = VehicleFactory.new
+
+    expect(factory).to be_instance_of(VehicleFactory)
+  end
+
+  it 'creates vehicles' do
+    factory = VehicleFactory.new
+    wa_ev_registrations = DmvDataService.new.wa_ev_registrations
+
+    expect(factory.create_vehicles(wa_ev_registrations)).to be_instance_of(Array)
+    expect(factory.create_vehicles(wa_ev_registrations).first).to be_instance_of(Vehicle)
+    expect(factory.create_vehicles(wa_ev_registrations).first.vin).not_to be(nil)
+    expect(factory.create_vehicles(wa_ev_registrations).first.year).not_to be(nil)
+    expect(factory.create_vehicles(wa_ev_registrations).first.make).not_to be(nil)
+    expect(factory.create_vehicles(wa_ev_registrations).first.model).not_to be(nil)
+  end
+
+end
```

Comment:
Great work with testing here. This one was a bit tricky to figure out how to test it. There's a lot of different approaches you could take on it, but I really like that you're making sure that it's an array, that it's made of Vehicle objects, and that the data isn't empty.